src/services/atomic-swap.service.js

```diff
@@ -50,10 +52,70 @@ const serviceCreate = async (partialTx, password) => {
     throw new Error('Unable to create the proposal on the Atomic Swap Service');
   }
 
+  await addListenedProposal(walletId, id, password);
   return { proposalId: id };
 };
 
+/**
+ * Adds a new proposal to the `listenedProposals` map of a wallet.
+ * @param {string} walletId
+ * @param {string} proposalId
+ * @param {string} password
+ * @return {Promise<void>}
+ */
+const addListenedProposal = async (walletId, proposalId, password) => {
```

Comment:
The `async` is because of the ws code that will be added?

Reply:
That's correct! This method will make a request to the backend, and so will need the async.